dlt/pipeline/trace.py

```diff
@@ -280,8 +284,8 @@ def end_trace_step(
     resolved_values = map(
         lambda v: SerializableResolvedValueTrace(
             v.key,
-            v.value,
-            v.default_value,
+            _mask_secret(v.value) if is_secret_hint(v.hint) else v.value,
```

Comment:
Do you think it could be helpful to be able to distinguish between (i) `None`, meaning that no value is set, and (ii) "*****" meaning that a secret is present?